connector_flow/task/abstract_task.py

```
@@ -96,7 +96,10 @@ def run(self, chunk_id=None, **kwargs):
         except:
             raise
         finally:
-            chunk.write({'state': new_state})
+            try:
+                chunk.write({'state': new_state})
+            except:
+                pass
```

I think this is not correct. If an error occurs during writing the state to the chunk I think the whole transaction will roll back, so it's missleading to silently catch the exeption here.

---

True, but now if any error happens in read_chunk(), final error is not what happen there, but error related with failed chunk write. In such case it's hard to understand why read_chunck() failed (found out it while writing unit tests).